docs/contrib/modifications.md

```diff
@@ -115,3 +115,31 @@ cd {{ config.extra.brand | lower }}-docs
 export DOCS_ENABLE_HTMLPROOFER=false
 tox -e serve
 ```
+
+## A note on commit messages
+
+When submitting a change, you will need to provide a commit message,
```

Comment:
Drop the first comma, maybe?

Reply:
Nope, but on second thought phrasing this as "When you submit a change" is better. I've updated the PR.